docs/development.md

```diff
@@ -116,7 +116,7 @@ Documentation how to check your code for coding style issues can be found [here]
 
 ## Thunder Travis CI
 
-All Thunder pull requests are execute on [Travis CI](https://travis-ci.org/BurdaMagazinOrg/thunder-distribution). On every pull request tests will be executed (or when new commits are pushed into pull request branch). Tests are executed against PHP versions 5.6 (with drush make install) and 7.2 (with composer install). All code will be checked against coding style.
+All Thunder pull requests are execute on [Travis CI](https://travis-ci.org/BurdaMagazinOrg/thunder-distribution). On every pull request tests will be executed (or when new commits are pushed into pull request branch). Tests are executed against PHP versions 5.6 (with drush make install) and 7.3 (with composer install). All code will be checked against coding style.
```

Comment:
This comment is not correct anymore, we do not test against 5.6 for a while now.